modules/channel-web/src/views/lite/components/messages/renderer/QuickReplies.tsx

```diff
@@ -15,13 +15,19 @@ import { Button } from './Button'
  * @return onSendData is called with the reply
  */
 export class QuickReplies extends Component<Renderer.QuickReply> {
+
+  componentDidMount() {
+    this.props.disableFreeText ? this.props.store.composer.lock() : this.props.store.composer.unlock()
```

Comment:
`this.props.store.composer.setLocked(this.props.disableFreeText)`

Reply:
@EFF Thanks for your comment, i think there is an issue in unit tests of NLU i searched in other PRs they got same issue. 

@allardy thanks for your review i'll do changes

Reply:
@allardy @EFF Done